recipes/mu-repo/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "mu_repo" %}
```

Comment:
Directory needs to be renamed to match the recipe for conversion of this to complete. Could you please send a follow-up PR to correct this, @nicoddemus? Feel free to add `[ci skip]` to the commit message as we already know the recipe works. Thanks.